api/version/version.go

```diff
@@ -0,0 +1,29 @@
+package version
+
+import "fmt"
+
+var (
+	GitCommit  = ""
+	APIVersion Version
+)
+
+func init() {
+	commit := GitCommit
+	if commit == "" {
+		commit = "unknown"
+	}
+
+	APIVersion = Version{
+		Data: struct {
+			Version string `json:"version"`
+		}{
+			Version: fmt.Sprintf("Blob Archiver API/%s", commit),
+		},
+	}
+}
+
+type Version struct {
```

Comment:
Since you already depend on the optimism monorepo, you might as well use the `op-service/eth.APIVersionResponse` and `VersionInformation` types.

Reply:
Will do -- thank you!